database/migrations/2020_10_15_000300_create_cart_restriction.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Igniter\Coupons\Database\Migrations;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCartRestriction extends Migration
+{
+    public function up()
+    {
+        Schema::table('igniter_coupons', function (Blueprint $table) {
+            $table->boolean('is_limited')->default(false);
```

Comment:
From the database schema, it's hard to tell what value this column holds. Maybe `is_limited_to_cart_item` or `applies_to_cart_item`? Sorry about the back and forth ;)